CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version [3.3.1] - (2019-04-25)
+- Renamed `branch()` method to `clone()`. The old `branch()` method still exists to maintain backward compatibility 
+until next major release and has @Deprecated annotation. 
```

Comment:
I like the start of it but the rest of the comments in the file are rather short.

What do you think about something like

```
- Created `.clone()` method of environments, to be used instead of `.branch()`.
- Deprecated `.branch()` of environments.
```

:thinking:

Reply:
Yes, I live the brevity.